doc/en/user/source/community/ogc-api/configure.rst

```diff
@@ -0,0 +1,49 @@
+Configuring the GeoServer OGC API module
+========================================
+
+The OGC API module is mostly using the same configurations as the equivalent OWS services.
+So, for example, setting the WFS limited SRS list will also limit the SRS list for OGC API - Features.
+
+The OGC API module is also using the same security configuration as the equivalent OWS services.
+
+In addition, the OGC API module has some unique configuration options, explained below.
+
+Custom links for the "collections" resource
+-------------------------------------------
+
+By specification, the ``collections`` resource can have a number of additional links, beyond
+the basic ones that the service code already includes. 
+
+The administrator can add links for this resource either in the Global pages, or in the
+workspace specific settings:
+
+  .. figure:: img/links.png
+ 
+     Links editor
+
+Link editor column description:
+
+* **rel**: the link relation type, as per the OGC API - Features specification
+* **Mime type**: the mime type for the resource found following the link
+* **URL**: the link URL
+* **Title**: the link title (optional)
+* **Service**: the service for which the link is valid (optional, defaults to all) 
+
+
+Common links relationships that could be added for the ``collections`` resource are:
```

Comment:
Even though, they refer to the same concept, I think this should be "link relations"

Reply:
Quoting from OGC tutorials:

---
The server identifies the relationship between a resource and other linked resources through a link relation type, represented by the attribute ‘rel’.
---

So yep, "relation" it is, indeed!